app/src/main/java/com/example/trempel/network/ServiceProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.trempel.network
+
+import com.example.trempel.network.service.RetrofitService
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+internal object ServiceProvider {
+
+    private const val BASE_URL = "https://fakestoreapi.com/"
+    private var retrofit: Retrofit? = null
```

Comment:
do we really need `retrofit` defined here?

Reply:
done